src/NServiceBus.Testing.Tests/Handler/ExpectDeferTests.cs

```diff
@@ -17,26 +17,6 @@ public void ShouldAssertDeferWasCalledWithTimeSpan()
                 .OnMessage<ITestMessage>();
         }
 
-        [Test]
-        public void ShouldAssertDeferWasCalledWithDateTime()
```

Comment:
So this is no longer needed?

Reply:
No longer needed, DateTimeOffset doesn't have any issues with converting to from different offsets because the offset property as part of the type is used for that.

We could keep the test but it then is expected to not fail. Not sure if that has value.